controllers/util/k8sutil.go

```diff
@@ -166,3 +166,15 @@ func GenerateNameForNonResilientCephBlockPoolSC(initData *ocsv1.StorageCluster)
 	}
 	return fmt.Sprintf("%s-ceph-non-resilient-rbd", initData.Name)
 }
+
+func GetStorageClusterInNamespace(ctx context.Context, cl client.Client, storageClusterNamespace string) (*ocsv1.StorageCluster, error) {
```

Comment:
function name already says `Namespace` and last arg is repeating the same, remove a duplicate from one of these? `GetStorageCluster` or `namespace string`?

Reply:
thinking again, suggestions may just show up function name and in that scenario as we are using in multiple places, `GetStorageClusterByNamespace` would be good?